metrics/metrics.go

```diff
@@ -48,9 +48,29 @@ var (
 		},
 	)
 
+	DeschedulerLoopDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "descheduler_loop_duration_seconds",
+			Help:           "Time taken to complete a strategy descheduling operation",
+			StabilityLevel: metrics.ALPHA,
+			Buckets:        []float64{0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 100, 250, 500},
+		}, []string{})
+
+	DeschedulerStrategyDuration = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "descheduler_strategy_duration_seconds",
+			Help:           "Time taken to complete Each strategy of the descheduling operation",
+			StabilityLevel: metrics.ALPHA,
+			Buckets:        []float64{0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, 50, 100},
+		}, []string{"strategy", "profile"})
```

Comment:
wondering how useful the `profile` label is here. I suppose different profiles can have different strategy options, affecting the run time of each. In that case though, maybe it could be useful in the overall cycle latency metric as well.

Reply:
There is one tracked for the entire loop and one for each strategy and profile combo. Do you want to track one for each profile as well ?

Reply:
Right now we track to things in this PR. One that tracks the entire loop itself. And one that tracks the duration for each strategy and profile combination. We can track one for just the profile itself as well may be ? That will give you breakdown of the loop vs profile, and profile vs strategy as well. @damemi

Reply:
Ah I see. In the future, I think it could be useful to break the full loop duration down by profile as well, but I suppose if you're trying to debug why one profile takes longer than another then you really are looking for the strategy breakdown anyway. This looks good to me